### PR TITLE
Thomas williams/hello world

### PR DESCRIPTION
**Pull request by the-arkhive**
Fixes Issue #1  by making a basic hello world page to build the app on top of.

### CODE REVIEW

File: docs/notes.txt [hunk 1]
```diff
@@ -0,0 +1,196 @@
+- when you call a class from an HTML block is displays the `render` portion of that class
```
Comment (EverlastingBugstopper) [inline docs/notes.txt]: Did you mean to commit this file?

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
   render() {
     return (
       <div className="App">
         <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
+          <h1>Who Twote That </h1>
           <p>
-            Edit <code>src/App.js</code> and save to reload.
+            A Twitter game to see how well you know your friends, or really any two Twitter users for that matter. I'm using this to learn <a href="https://reactjs.org/tutorial/tutorial.html">React.js</a>.
           </p>
```
Comment (EverlastingBugstopper) [inline src/App.js:15]: Any idea why git hates these tabs? Would recommend using `prettier` to get auto-formatting down so you don't have to worry about Windows doing weird things to your files

**Review by EverlastingBugstopper — APPROVED** [review]
LGTM

File: docs/notes.txt (hunk 1 above)
Comment (the-arkhive) [reply docs/notes.txt]: No, I need to add the docs folder to the git ignore, will do that then update the branch.

File: src/App.js (hunk 2 above)
Comment (the-arkhive) [reply src/App.js:15]: I already have it installed actually, so I will run it and hopefully it fixes things.

**Pull request merged**